doc/overview.md

```diff
@@ -128,9 +128,11 @@ API token set up locally to be able to submit the results.
 
 ### Email Generator
 
-The Email Generator step listens for pub/sub events for when tests are complete
+The Email Generator step listens for pub/sub events when tests are complete
 to then generate a summary and send it via email.  The current implementation
-only prints the summary in the logs, and generates one for every test results
-received.  A production solution would send emails to various recipients
-depending on the tests and would wait for an event signalling that a group of
-tests has completed (or timed out).
+renders an email message based on the test report jinja template, opens a conenction
+to a STMP server and sends a report message to a user or a list of recipients once
+a change happened in the test results. At this moment emails are sent in case of a
+test fail or pass. A complete production solution would send emails to various
```

Comment:
```suggestion
test fails or passes. A complete production solution would send emails to various
```

Reply:
in this case fail and pass are nouns, no? To use fails or passes we need to replace "in case of" with "in case" as well.

Reply:
I think in both cases, we need `fails or passes`. We should have @gctucker's review for this.

Reply:
To be honest I'm not sure I understand the meaning of the sentence. Do you mean that email reports are sent either when a test is passing or failing? Or do you mean that the reports contain information about tests that passed and failed?

Reply:
At this moment this sentence doesn't make sense anymore..The reports is not sent in case of a test fail or pass. :)